pages/external-redirect.js

```diff
@@ -84,7 +84,7 @@ const ExternalRedirectPage = () => {
                   <FormattedMessage id="actions.continue" defaultMessage="Continue" />
                 </StyledButton>
               </a>
-              <Link route={fallback} onClick={() => setPendingAction('CANCEL')}>
+              <Link href={`/${fallback}`} onClick={() => setPendingAction('CANCEL')}>
```

Comment:
The one in https://github.com/opencollective/opencollective-frontend/blob/915be5e17ab6df878375e25c8c56248a4de4275a/components/contribution-flow/index.js#L232 already has a `/`, and the ones we generate from https://github.com/opencollective/opencollective-api/blob/a759154e1efcb5ef1c196f772613e7c0ba8bea4d/server/lib/sanitize-html.ts#L187 have an `https://` prefix.
```suggestion
              <Link href={fallback} onClick={() => setPendingAction('CANCEL')}>
```